meta/set_auto.go

```diff
@@ -255,6 +260,17 @@ func (s Set) Dart(station string) (Dart, bool) {
 	return Dart{}, false
 }
 
+// Domain is a helper function to return a Domain value and true if one exists.
+func (s Set) Domain(name string) (Domain, bool) {
+	for _, v := range s.domains {
+		if name != v.Name {
+			continue
+		}
+		return v, true
+	}
+	return Domain{}, false
```

Comment:
suggestion: have you considered `slices` package, might be more concise...(7 LOC later)...nevermind 😑